### DISCUSSION TITLE
Regarding Stripe Capture(Saleor 2.11)

### DISCUSSION THREAD

**Discussion by KrEnjuTSU** [saleor/saleor#8009] [Q&A]
Hello,

I am trying to do a partial capture based on partial fulfillment. Initially, I  disable Automatic payment capture in Stripe plugin settings. Then I am using orderCapture mutation to capture the amount based on the fulfillment. But the orderCapture is not working properly, the full amount is captured always irrespective of the amount I am passing in orderCapture mutation. 
eg: I authorize the card for 50 euros, I try to capture 40 euros but it does not work like that, by default the entire pre-authorized amount is getting captured when I am passing 40  euros using orderCapture mutation. Can anyone help me with this?

**Comment by karolkielecki** [comment]
@KrEnjuTSU Hi there! I recommend you upgrade to Saleor 3. We've rebuilt the Stripe plugin and also added many new great features, including multichannel. It's still a beta version, but it's better than 2.11
